acceptance/tests/wan-federation/wan_federation_test.go

```diff
@@ -43,10 +43,6 @@ func TestWANFederation(t *testing.T) {
 			env := suite.Environment()
 			cfg := suite.Config()
 
-			if cfg.UseKind {
-				t.Skipf("skipping wan federation tests as they currently fail on Kind even though they work on other clouds.")
-			}
```

Comment:
I don't think I've actually fixed anything here, so I'm curious if this is still broken in CI.